crux-core/src/crux/api.clj

```diff
@@ -155,16 +156,16 @@
   Returns a map with details about the submitted transaction,
   including tx-time and tx-id.")
 
-  (open-tx-log ^crux.api.ITxLog [this from-tx-id with-ops?]
+  (open-tx-log ^java.io.Closeable [this from-tx-id with-ops?]
     "Reads the transaction log. Optionally includes
   operations, which allow the contents under the :crux.api/tx-ops
   key to be piped into (submit-tx tx-ops) of another
   Crux instance.
 
-  from-tx-id      optional transaction id to start from.
-  with-ops?       should the operations with documents be included?
+  from-tx-id  optional transaction id to start from.
+  with-ops?   the operations with documents be included?
```

Comment:
nit: 'include operations with documents?'.
Unsure on that one, just reads a bit strange to me.

Reply:
yep, unintentional delete, derp - thanks